### ISSUE TITLE
about the code and the paper

### ISSUE THREAD

**Issue by jingubang1219** [pencoa/PCNN#8]
hi,firstly thxs for you resource~
1. what i wonder is i can't find where is the part of the raw paper mentioned "Multi-instance Learning", is it that you still use the aver of the bag loss to train the model?
2.in the resource ,you first slice the vector into pieces because  the tensorflow doesn't support slice tensor,could i first embedding and the use mask to slice the tensor?

**Comment by jingubang1219** [comment]
o,i'm sorry,i find the where the "Multi-instance Learning" is ,thxs~
